### PR TITLE
LPS-105689 View active changelist regardless of context in DXP

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/servlet/taglib/ChangeTrackingIndicatorDynamicInclude.java [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+		CTPreferences ctPreferences =
+			_ctPreferencesLocalService.fetchCTPreferences(
+				themeDisplay.getCompanyId(), 0);
+
+		if (ctPreferences == null) {
+			return;
+		}
+
+		try {
+			PageContext pageContext = PageContextFactoryUtil.create(
```
Comment (Preston-Crary) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/servlet/taglib/ChangeTrackingIndicatorDynamicInclude.java]: Don't need the PageContext, call httpServletResponse.getWriter().

Comment (Preston-Crary) [reply modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/servlet/taglib/ChangeTrackingIndicatorDynamicInclude.java]: Unless it's needed by ReactRenderer?

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/servlet/taglib/ChangeTrackingIndicatorDynamicInclude.java [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+
+			String componentId =
+				_portal.getPortletNamespace(CTPortletKeys.CHANGE_LISTS) +
+					"IndicatorComponent";
+			String module =
+				_npmResolver.resolveModuleName("change-tracking-web") +
+					"/dynamic_include/ChangeTrackingIndicator";
+
+			ComponentDescriptor componentDescriptor = new ComponentDescriptor(
+				module, componentId, null);
```
Comment (Preston-Crary) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/servlet/taglib/ChangeTrackingIndicatorDynamicInclude.java]: Call the two arg constructor and inline ComponentDescriptor.

**Comment by Preston-Crary** [comment]
Can you push those two changes? Other than that looks good.

**Comment by stsquared99** [comment]
@Preston-Crary updated

**Comment by Preston-Crary** [comment]
ci:test:upgrade

**Comment by Preston-Crary** [comment]
Please see https://github.com/stsquared99/liferay-portal/pull/106

**Pull request closed without merging**